multinet/db.py

```diff
@@ -608,3 +610,48 @@ def count_text(filt: str) -> str:
         "edges": list(aql_query(workspace, query)),
         "count": next(aql_query(workspace, count)),
     }
+
+
+@lru_cache(maxsize=1)
+def uploads_database() -> StandardDatabase:
+    """Return the database used for storing multipart upload collections."""
+    sysdb = db("_system")
+    if not sysdb.has_database("uploads"):
+        sysdb.create_database("uploads")
+    return db("uploads")
+
+
+def create_upload_collection() -> str:
+    """Insert empty multipart upload temp collection."""
+    uploads_db = uploads_database()
+    upload_id = f"u-{uuid4().hex}"
+    uploads_db.create_collection(upload_id)
+    return upload_id
+
+
+def insert_file_chunk(upload_id: str, sequence: str, chunk: str) -> str:
+    """Insert b64-encoded string `chunk` into temporary collection."""
+    uploads_db = uploads_database()
+    if not uploads_db.has_collection(upload_id):
+        raise UploadNotFound(upload_id)
+
+    collection = uploads_db.collection(upload_id)
+
+    if collection.get(sequence) is not None:
+        raise AlreadyExists("Document", sequence)
```

Comment:
The resource that already exists is a chunk, not just a generic document. I suggest:

```suggestion
        raise AlreadyExists("Upload Chunk", `${upload_id}/${sequence}`)
```

Reply:
(I am still recommending this change, despite dismissing the review it is part of.)

Reply:
The string interpolation here is wrong. This is js style, not python